tabs/mission_control.js

```diff
@@ -2110,13 +2119,10 @@ TABS.mission_control.initialize = function (callback) {
                 P3Value = selectedMarker.getP3();
                 
                 if (disableMarkerEdit) {
-                    changeSwitchery($('#pointP3Alt'), TABS.mission_control.isBitSet(P3Value, 0));
+                    changeSwitchery($('#pointP3Alt'), TABS.mission_control.isBitSet(P3Value, MWNP.P3.ALT_TYPE));
                 }
 
-                P3Value = TABS.mission_control.setBit(P3Value, 0, $('#pointP3Alt').prop("checked"));
-                selectedMarker.setP3(P3Value);
-
-                P3Value = TABS.mission_control.setBit(P3Value, 0, $('#pointP3Alt').prop("checked"));
+                P3Value = TABS.mission_control.setBit(P3Value, MWNP.P3.ALT_TYPE, $('#pointP3Alt').prop("checked"));
```

Comment:
Line 2131: this needs to compare the BIT 0 value of P3 on entering the function (line 2119) with the BIT 0 value after updating with the state of the checkbox. So P3Value can't be redefined on line 2125, needs to be called something else.

`P3Value != selectedMarker.getP3()` will always be false as this is now although it won't work anyway because it's not checking BIT 0 specifically.

Reply:
I've moved the storage of the newly calculated P3 to after the check, if they are different.